src/tests/conftest.py

```diff
@@ -0,0 +1,51 @@
+import os
+from typing import Generator
+
+import pytest
+from flask import Flask
+from flask.testing import FlaskClient
+from gobstuf.api import get_flask_app
+from gobcore.secure.request import ACCESS_TOKEN_HEADER
+
+BASE = 'tests'
+
+
+def _env(var: str) -> str:
+    return os.environ[var]
+
+
+def _read_file(path: str) -> str:
+    with open(path, mode='r') as file:
+        return file.read()
+
+
+@pytest.fixture
+def app() -> Generator[Flask, None, None]:
+    """Creates a flask test app, with an app context.
+
+    The app can be used to create a test http client as well:
+    with app.test_client() as client:
+        client.get(...)
+    """
+    app = get_flask_app()
+    with app.app_context():
+        yield app
+
+
+@pytest.fixture
+def client(app: Flask) -> Generator[FlaskClient, None, None]:
+    with app.test_client() as client:
+        client.testing = True
+        yield client
+
+
+@pytest.fixture
+def jwt_header() -> dict:
+    return {ACCESS_TOKEN_HEADER: _read_file(f'{BASE}/utils/jwt.txt')}
+
+
+@pytest.fixture(autouse=True)
+def stuf_310_response(requests_mock):
```

Comment:
Which test uses this? Is autouse=True set because it is used in a "traditional" TestCase-test?

Reply:
Just the 1 test in this pr for the moment. I thought every request needs to be mocked with this response, so we don't have to specify it in the function arguments.